### ISSUE TITLE
Usage of deprecated function

### ISSUE THREAD

**Issue by Ramblurr** [iamruinous/plugin-powerline#2]
When using this plugin you'll get the following:

```
warning: function available is deprecated and will be removed soon.
called on line 2 of file ~/.local/share/omf/pkg/powerline/init.fish
warning: function available is deprecated and will be removed soon.
called on line 9 of file ~/.local/share/omf/pkg/powerline/init.fish
```

**Comment by Ramblurr** [comment]
fyi, replacing `available` with `type -q` fixed it.

**Comment by temsa** [comment]
@iamruinous any ETA for fixing the issue, do you want a PR ?

You're plugin rocks BTW ;)

**Comment by iamruinous** [comment]
Fixed in #3
